# MA0068 - Invalid parameter name

````csharp
class Test
{
    [System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("unknown")] // non-compliant, "unknown" is not a parameter of the method
    public void A(string a) { }

    [System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("a")] // compliant, "a" is not a parameter of the method
    public void A(string a) { }
}
````
